specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/FirewallPolicyRuleCollectionGroupWithWebCategoriesGet.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
this should go in the examples folder

Reply:
These were redundant files. I have removed them.